ggshield/iac/iac_scan_models.py

```diff
@@ -37,8 +45,62 @@ class IaCDiffScanResult(Base):
     IaCDiffScanResult, BaseSchema
 )
 
+logger = logging.getLogger(__name__)
+
 
 class MockClient(GGClient):
```

Comment:
Sorry, I did not picked it on the first reviews but having mock in the name of the class and the methods seems very counter intuitive to me.
I would expect a mock to be a fake version of the object and not make a "real" request.